selfdrive/car/card.py

```diff
@@ -174,7 +174,8 @@ def controls_update(self, CS: car.CarState, CC: car.CarControl):
       # send car controls over can
       now_nanos = self.can_log_mono_time if REPLAY else int(time.monotonic() * 1e9)
       self.last_actuators_output, can_sends = self.CI.apply(CC, now_nanos, self.frogpilot_toggles)
-      self.pm.send('sendcan', can_list_to_can_capnp(can_sends, msgtype='sendcan', valid=CS.canValid))
+      result = can_list_to_can_capnp(can_sends, msgtype='sendcan', valid=CS.canValid)
+      self.pm.send('sendcan', result)
```

Comment:
Oops, this is unnecessary. This was part of me debugging why can_list_to_capnp was failing (it was the missing `t` value in `secoc.py`)